### ISSUE TITLE
pds graph should contain the branch instruction (and maybe conditional) of each bb

### ISSUE THREAD

**Issue by radare** [radare/radare2#13296]
otherwise we cant understand if its a ret, conditional jmp or wat

comments on this?

**Comment by radare** [comment]
![screenshot 2019-03-06 at 18 46 05](https://user-images.githubusercontent.com/917142/53901852-3868fd00-4040-11e9-8017-acfa55a4eb94.png)

**Comment by XVilka** [comment]
Agree. Or we can think on how to differentiate them without adding line with branching instruction.
